Makefile

```diff
@@ -78,7 +78,7 @@ calico_containers/calico-node.tar: caliconode.created
 	docker save --output calico_containers/calico-node.tar calico/node
 
 st: binary calico_containers/busybox.tar calico_containers/routereflector.tar calico_containers/calico-node.tar run-etcd
-	dist/calicoctl checksystem --fix
+	sudo dist/calicoctl checksystem --fix
```

Comment:
`make` commands should not be forced to run as root, however this will cause the user to be prompted unless they have sudoers `NOPASSWD` or the like.